Doc/distributing/index.rst

```diff
@@ -1,12 +1,11 @@
 .. _distributing-index:
 
-###############################
-  Distributing Python Modules
-###############################
+****************************
+Distributing Python Packages
+****************************
```

Comment:
Hello!  Thanks for making a PR to improve Python docs!

Distutils is deprecated: its code and docs have been adopted by the setuptools project.  Distutils and setuptools are also less important now that pyproject.toml exists so that people can use different build tools and keep compatibility with PyPI and pip / tox / etc.

For these reasons, I would only accept clear fixes, not formatting or style changes.  The two changes here for example (header style and word choice) are out of scope.